Added Support for Microsoft Graph Single User integration (#30967)
demisto/content: Packs/EmailCommunication/ReleaseNotes/2_0_15.md

#### Scripts

##### PreprocessEmail

> - Added support for getting attachments during preprocessing email for Microsoft Graph Mail Single User.

Reviewer: ```suggestion
- Added the ability to get attachments during preprocessing email for **Microsoft Graph Mail Single User** integration.
```
